pkg/cri/cri.go

```diff
@@ -33,7 +34,12 @@ func New() *Client {
 }
 
 func (c *Client) GetRunningPodSandboxes() (map[string]*SandboxInfo, error) {
-	conn, err := grpc.Dial(criSocketPath, grpc.WithInsecure())
+	socketPath := dockerSocketPath
+	if info, err := os.Stat("/var/run/cri.sock"); err == nil && !info.IsDir() {
+		socketPath = criSocketPath
+	}
+	log.Debugf("Getting running pod sandboxes from %q", socketPath)
+	conn, err := grpc.Dial(socketPath, grpc.WithInsecure())
```

Comment:
I suppose we can turn this into a blocking call (WithBlock() with a backoff timeout) in a future commit...